backend/tests/fixtures/common.py

```diff
@@ -0,0 +1,30 @@
+import base64
+from typing import Any, Callable
+
+import pytest
+
+from hct_mis_api.apps.account.fixtures import BusinessAreaFactory
+
+
+@pytest.fixture()
+def afghanistan(db: Any) -> BusinessAreaFactory:
```

Comment:
Where db is used?

Reply:
It is needed to access the database
[documentation](https://pytest-django.readthedocs.io/en/latest/helpers.html#db)

Reply:
ok, clear